### DISCUSSION TITLE
[Feature] Add the ability to assign people to groups

### DISCUSSION THREAD

**Discussion by imler** [immich-app/immich#8678] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have a library with many people because I have many old photos of my whole family. On top of that, there are friends, coworkers, etc. I would like to be able to organize all those people into smaller groups.

I would like to assign people into groups - Friends, Family, Work, etc. Everyone could create as many categories as they want with chosen specific names.

Would be great if each category could have subcategories, so it would be possible to further split the family group into smaller more specific groups.

The important thing is that one person can belong to several groups and Immich should allow that. Imagine a situation where someone from a family is also working in the same company.

Such an organization would dramatically declutter the "People" page and allow for easier finding of specific people.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#6079. Please actually search for duplicates, don't just check the box ;)
